lunes_cms/static/js/toggle_document_form_fields.js

```diff
@@ -12,7 +12,7 @@ $(document).ready(() => {
   $("#id_word_type").trigger("change");
 });
 
-//Toggle the drop down for grammatical gender depending on the chosen word type
+// Toggle the drop down for grammatical gender depending on the chosen word type
```

Comment:
```suggestion
```
I needed the comment to have an easy time to understand what the function does. In `Clean Code` it says that this is a code smell. Instead we should maybe re-write the function to make it a more self-explanatory. Please see my suggestion for it below.
However I'm aware, that this partly comes down to personal preference or in which school of thought one was taught, so I would also understand if we keep it this way :)